metrics/web/docs/features/firebase_analytics/01_firebase_analytics_design.md

```diff
@@ -0,0 +1,45 @@
+# Firebase Analytics design
+
+## Metrics application
+
+### Domain layer
+
+In the application domain layer, we should add an ability to log user activities. For this purpose, we should: 
+
+1. Add the `logLogin` and `logPageView` methods to the `AnalyticsRepository`.
+2. Implement the `LogLoginUseCase` and `LogPageViewUseCase` classes.
+3. Add the needed parameter classes.
+
+So, the domain layer should look like this:
+
+![Domain layer diagram](http://www.plantuml.com/plantuml/proxy?cache=no&fmt=svg&src=https://github.com/platform-platform/monorepo/raw/firebase_analytics_design/metrics/web/docs/features/firebase_analytics/diagrams/firebase_analytics_domain_class.puml)
```

Comment:
Would it be great to explain a bit about what is `PageName` class? Also, maybe it should be noted as a class in a class diagram because I think it won't be an enum, but it will be a class that has similar behavior to an enum?